ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -306,6 +307,13 @@ public void setInjectedJavaScript(WebView view, @Nullable String injectedJavaScr
     ((ReactWebView) view).setInjectedJavaScript(injectedJavaScript);
   }
 
+  @ReactProp(name = "mixedContentModeEnabled")
+  public void setMixedContentModeEnabled(WebView view, boolean enabled) {
+    if (enabled && Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+      view.getSettings().setMixedContentMode(WebSettings.MIXED_CONTENT_COMPATIBILITY_MODE);
```

Comment:
Sorry looking at the Android API more closely, I think we want to expose the underlying API: always, never, compatibility as enums.